### PR TITLE
ChargeDistribution

### PR DESCRIPTION
**Pull request by ahmedhamila**

### CODE REVIEW

File: src/pages/charge/chef-departement/ConsultationEnseignant/ConsultationEnseignant.css
Comment (stormynight9) [inline src/pages/charge/chef-departement/ConsultationEnseignant/ConsultationEnseignant.css:1]: use css modules besh styles mta3ek ye5dmou ken fel components mta3ek w may2athrouch 3la 5edmet nes lo5rin

File: index.html [hunk 1]
```diff
@@ -11,5 +11,8 @@
     <body>
         <div id="root"></div>
         <script type="module" src="/src/main.tsx"></script>
+        <script>
+            const global = globalThis
```
Comment (stormynight9) [inline index.html:15]: I'm guessing you're not using this, if so delete it.

File: src/pages/charge/chef-departement/GestionDiplomes/DiplomePathing/DiplomePathing.jsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+const DiplomePathing = () => {
+    const [diplomes, setDiplomes] = useState([])
+    const [selectedDiplome, setSelectedDiplome] = useState(null)
+    const [selectedSection, setSelectedSection] = useState(null)
+    const [selectedNiveau, setSelectedNiveau] = useState(null)
+    const [selectedSemestre, setSelectedSemestre] = useState(null)
+    const location = useLocation()
+    useEffect(() => {
+        const getDiplomes = async () => {
+            const response = await fetch("http://localhost:8090/api/isimm/distributionCharge/diplome", {
```
Comment (stormynight9) [inline src/pages/charge/chef-departement/GestionDiplomes/DiplomePathing/DiplomePathing.jsx:18]: mat7otesh `http://localhost:8090` instead use the env variable `import.meta.env.VITE_API_URL`

**Review by stormynight9 — COMMENTED** [review]
fama barcha warnings 7awel na7ehom besh matgalegsh nes lo5rin, homa mostly mta3 hajet unused.
fama zeda conflicts

File: index.html (hunk 1 above)
Comment (ahmedhamila) [reply index.html:15]: I'm using a module eli 7echtou bel reference heki , i can't delete it .
does it harm other parts?

File: src/pages/charge/chef-departement/GestionDiplomes/DiplomePathing/DiplomePathing.jsx (hunk 2 above)
Comment (ahmedhamila) [reply src/pages/charge/chef-departement/GestionDiplomes/DiplomePathing/DiplomePathing.jsx:18]: I know i'll change them juste 5ater ma3 b9yet l'equipe 3arfouch ye5dmou beha .

File: .env
Comment (stormynight9) [inline .env:1]: file name should be .env.local elli hiya mawjouda fel .gitignore

Comment (ahmedhamila) [reply .env:1]: Done

File: src/pages/charge/chef-departement/ConsultationEnseignant/ConsultationEnseignant.css
Comment (ahmedhamila) [reply src/pages/charge/chef-departement/ConsultationEnseignant/ConsultationEnseignant.css:1]: Done

**Pull request merged**